namespaces/live.cloud-platform.service.justice.gov.uk/c100-application-staging/resources/s3.tf

```diff
@@ -0,0 +1,196 @@
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+module "s3_bucket" {
+
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-s3-bucket?ref=4.7.1"
+  team_name              = var.team_name
+  business-unit          = var.business_unit
+  application            = var.application
+  is-production          = var.is_production
```

Comment:
@WillTaylor22 , can u update the environment and is_production to match the names definied in variables.tf https://github.com/ministryofjustice/cloud-platform-environments/blob/main/namespaces/live.cloud-platform.service.justice.gov.uk/c100-application-staging/resources/variables.tf#L5-L11